src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -542,12 +542,9 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
-    partnerInquirerCollectorProfileLoader: gravityLoader<
```

Comment:
should we remove this endpoint from Gravity as well? Doesn't seem to be used anywhere but here.

Reply:
Sure! But there's no rush, once this PR is deployed, can check back on Datadog after a week to ensure this endpoint hasn't been hit any more.